alf/metrics/metrics.py

```diff
@@ -230,40 +238,54 @@ def call(self, time_step):
 
         is_first = time_step.is_first()
 
-        def _update_accumulator_(path, acc, val):
-            """In-place update of the accumulators."""
+        def _update_accumulator_(path, mask, acc, val):
+            """In-place update of the accumulators and mask."""
+            val_valid = torch.isfinite(val)
+            # If at any step the value is valid, then the acc value becomes valid
```

Comment:
This behavior should be clearly document in the doc-string of `AverageEpisodicAggregationMetric`

Reply:
> This behavior should be clearly document in the doc-string of `AverageEpisodicAggregationMetric`

Added.